frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -152,6 +152,20 @@ function FileManagerMenu:setUpdateItemTable()
                 },
             }
         }
+    elseif Device.isKindle() then
```

Comment:
I have not tested folder screensaver functoinality. Bessides there is no option to disable screensaver globally in Kobo.
Maybe in future I will "fix it".